pkgs/development/coq-modules/mathcomp-analysis/default.nix

```diff
@@ -35,8 +36,7 @@ let mca = mkCoqDerivation {
   };
 }; in
 mca.overrideAttrs (o:
-  let ext = { propagatedBuildInputs = o.propagatedBuildInputs
-                                      ++ [ hierarchy-builder ]; };
+  let ext = { propagatedBuildInputs = o.propagatedBuildInputs ++ [ hierarchy-builder ]; };
```

Comment:
Does it respect the magic 80-line limit? If not, there is no reason to merge the lines IMHO.

Reply:
Sure, just wasn't sure what the newline was there for.